lib/galaxy/model/__init__.py

```diff
@@ -204,8 +204,12 @@ def system_user_pwent(self, real_system_username):
             except KeyError:
                 pass
         else:
```

Comment:
@bernt-matthias  I think the default value for 'real_system_username' is 'user_email'. This function 'system_user_pwent' is called only when 'external_runjob_script' is set in galaxy.ini. Do I still need to check for None ?